pydis_site/apps/home/views/home.py

```diff
@@ -0,0 +1,110 @@
+import requests
+from django.shortcuts import render
+from django.utils import timezone
+from django.views import View
+
+from pydis_site.apps.home.models import RepositoryMetadata
+
+
+class HomeView(View):
+    """The view"""
```

Comment:
this docstring doesn't spark anything to me, how about something like "The index page of the website."

Reply:
oh lol I guess I must've started writing this and just forgotten about it.